content/telegraf/v1.15/administration/windows_service.md

```diff
@@ -8,41 +8,59 @@ menu:
     parent: Administration
 ---
 
-# Running Telegraf as a Windows service
+Telegraf natively supports running as a Windows service.
 
-Telegraf natively supports running as a Windows service. Outlined below are
-the general steps to set it up.
+The following commands are available:
 
-1. Obtain the Telegraf distribution for Windows.
-2. Create the directory `C:\Program Files\Telegraf` (if you install in a different location, specify the `-config` parameter with the desired location)
-3. Place the `telegraf.exe` and the `telegraf.conf` files into `C:\Program Files\Telegraf`.
-4. To install the service into the Windows Service Manager, run the following in PowerShell as an administrator. If necessary, you can wrap any spaces in the file directories in double quotes `"<file directory>"`:
-
-   ```
-   > C:\"Program Files"\Telegraf\telegraf.exe --service install
-   ```
+| Command                            | Effect                        |
+|------------------------------------|-------------------------------|
+| `telegraf.exe --service install`   | Install telegraf as a service |
+| `telegraf.exe --service uninstall` | Remove the telegraf service   |
+| `telegraf.exe --service start`     | Start the telegraf service    |
+| `telegraf.exe --service stop`      | Stop the telegraf service     |
 
-5. Edit the configuration file to meet your requirements.
+Outlined below are the general steps to install Telegraf as a Service.
 
-6. To verify that it works, run:
+{{% note %}}
+Installing a Windows service requires administrative permissions.
+Be sure to [launch Powershell as administrator](
+https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
+{{% /note %}}
 
+1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
+2. In PowerShell, run the following as an administrator:
+   ```powershell
+   > cd "C:\Program Files\InfluxData\Telegraf"
+   > .\telegraf.exe --service install --config "C:\Program Files\InfluxData\Telegraf\telegraf.conf"
    ```
+   When installing as service in Windows, always double check to specify full, correct path of the config file.
```

Comment:
When installing Telegraf as a Windows service, provide the **absolute path** of the Telegraf configuration file.